### PR TITLE
test(DIST-142): Embed functional test coverage

### PR DESCRIPTION
**Pull request by antoniofull**
Improves functional test on mobile.
- [x] Explain the **motivation** for making this change.
- [x] Provide a **test plan** demonstrating that the code is solid.
- [x] Match the **code formatting** of the rest of the codebase.
- [x] Two other colleagues must **approve** this change.
- [ ] **QA** have tested the solution provided.
- [ ] **Design and UX** must validate this change.

### CODE REVIEW

File: e2e/cypress-utils.js [hunk 1]
```diff
@@ -86,3 +83,26 @@ export const openAsMobile = (url) => {
     }
   })
 }
+
+export const setupEmbedOnMobile = () => {
+  if (Cypress.isBrowser('chrome')) { // inside of an iframe is testable only in Chrome
+    getIframeBody(IFRAME).find('[data-qa="start-button"]').should('have.text', 'Start').click()
+
+    cy.wait(1500).then(() => {
```
Comment (lfons) [inline e2e/cypress-utils.js:91]: Just a small comment. Maybe we could add these waiting times in a const variable so it is easy to adjust them at once if needed.

Comment (antoniofull) [reply e2e/cypress-utils.js:91]: makes sense, in some cases might not be needed but the main issue is the loading time of the iframe. Let me me have a look later and I'll move them in some variables

Comment (antoniofull) [reply e2e/cypress-utils.js]: fixed now thanks!

**Comment by antoniofull** [comment]
> Looking good. I would like to ask you one thing - can you use the same form, the one without welcome page (https://admin.typeform.com/to/Z9k3bK) for each test? That way it will be more obvious on mobile vs desktop. Thanks :)
on it!

**Pull request merged**